pyirf/scripts/make_DL3.py

```diff
@@ -0,0 +1,338 @@
+"""Script to produce DL3 data from DL2 data and a configuration file.
```

Comment:
DL3 data means normally an event list together with the IRFs. What does this script do?

We should probably have a script that calculates irfs, one that calculates sensitivity and one that creates a DL3 event list file from DL2 and irf files.

Reply:
it does everything

Reply:
> We should probably have a script that calculates irfs, one that calculates sensitivity and one that creates a DL3 event list file from DL2 and irf files.

This can be a refactoring proposal to add to #10

Reply:
> it does everything

It creates DL3 event lists? For which of the input files? How is it determined which events are used for IRF calculation and which are used for the event list?

Reply:
The DL3 list is made by the DL2 data that pass the best cut-offs and that is used to create the IRFs

It creates one file per particle type with the suffix "_processed" in HDF5 format

Reply:
I don't think that's a DL3 event list, it works only with simulated data and is only a by-product of the IRF production.
As a more general comment, it's not clear to me if py**irf** should produce the event lists. To be discussed elsewhere than here.

Reply:
It is not a DL3 list in the strictest sense (that would be "real" DL2 data to which the best cuts from the IRFs from pyirf are are applied)

in the case of pure simulated data pipeline (like the case we are approaching now) this data is the DL2 simulated data which passes the best cuts and therefore is used to create the IRFs

so, no I don't think we should create the DL3 event lists in this case (it is not clear to me when one would use those events in the end, am I missing something?)

should I rewrite the public docstring more specifically?

Reply:
this is now explained better in issue #30